public/index.html

```diff
@@ -1,54 +1,124 @@
+    <p>Work at the Mozilla Foundation or Mozilla Corporation and want to accept payments? This is for you.</p>
+  </div>
+
+  <div class="container">
+    <section>
+      <h2><a name="quick-example"></a>Quick Example</h2>
+      <p><a href="#top">↑ Back to top</a></p>
+      <p>We set up a sample an example product called "Mozilla Concrete". You can buy some for $10.</p>
+      <button type="button" class="btn btn-lg btn-success brick">Buy</button>
+    </section>
+
+    <section>
+      <h2><a name="features"></a>Features</h2>
+      <p><a href="#top">↑ Back to top</a></p>
+      <ul>
+        <li>Authentication through Firefox Accounts</li>
+        <li>Subscriptions</li>
+        <li>Management interface</li>
+      </ul>
+      <p>Todo list:</p>
+      <ul>
+        <li>One off purchases</li>
+        <li>Anonymous purchases</li>
+        <li>Paypal</li>
+      </ul>
+    </section>
+
+    <section>
+      <h2><a name="drop-in"></a>Drop-in UI</h2>
+      <p><a href="#top">↑ Back to top</a></p>
+      <p>The drop-in UI presents as an iframe popup, following the style of Firefox Accounts. To integrate you need to do a couple of simple steps.</p>
+      <p>Using "Mozilla Concrete" we set up two demo products:</p>
+      <h3>1. Configuration</h3>
+      <p><a href="http://payments.readthedocs.org/en/latest/design/api/configuration.html">Configuration</a> is done through a simple Python file:</p>
+      <pre>
+'mozilla-concrete': {
+  'email': 'support@concrete.mozilla.org',
+  'name': _('Mozilla Concrete'),
+  'url': 'http://pay.dev.mozaws.net/',
+  'terms': 'http://pay.dev.mozaws.net/terms/',
+  'products': [{
+    'id': 'mortar',
+    'description': _('Mortar'),
+    'amount': '5.00',
+    'img': 'http://bit.ly/mortar-png',
+  }]
+}
+      </pre>
+      <p><a href="http://payments.readthedocs.org/en/latest/design/api/configuration.html">read more →</a></p>
+
+      <h3>2. Button code</h3>
+      <p>Add in the <a href="http://payments.readthedocs.org/en/latest/design/api/client.html">button code</a> to trigger. In the above example, it listens to click events on a button and trigger the following:</p>
+      <pre>
+var client = new window.PaymentsClient({
+  accessToken: firefox_accounts_access_token,
+  product: {
+    id: 'mozilla-concrete-mortar',
+    image: 'http://bit.log/mortar-png' },
+});
+      </pre>
+      <p><a href="http://payments.readthedocs.org/en/latest/design/api/client.html">read more →</a></p>
+
+      <h3>3. Test it!</h3>
+      <p>The following connect to the sandbox where <a href="https://developers.braintreepayments.com/ios+ruby/reference/general/testing#credit-card-numbers">test credit cards</a> are accepted:</p>
+      <p><button type="button" class="btn btn-lg btn-primary brick">Brick for $10 a month</button></p>
+      <p><button type="button" class="btn btn-lg btn-primary mortar">Mortar for $5 a month</button></p>
+    </section>
+
+    <section>
+      <h2><a name="full-api"></a>Full API</h2>
```

Comment:
If you just want headings to be referenced with #full-api `<h1 id="full-api">Full API</h1>` will give you that you don't need the link.